src/themes/playful.css

```
@@ -334,14 +350,415 @@
+  }
+
+  code {
+    font-weight: var(--wa-font-weight-normal);
+    font-size: var(--wa-font-size-s);
+  }
+
+  img {
+    border-radius: var(--wa-corners-m);
+    box-shadow: var(--wa-shadow-level-1);
+  }
+
+  wa-rating {
+    --symbol-size: var(--wa-font-size-m);
+    --symbol-color: color-mix(in oklab, var(--symbol-color-active) 50%, transparent);
+  }
+
+  wa-switch {
+    --background: var(--wa-surface-default);
+    --border-width: var(--wa-form-controls-border-width);
+    --thumb-color: var(--wa-form-controls-resting-color);
+    --thumb-color-checked: var(--wa-color-base-95);
+    --thumb-size: 0.8em;
+    --height: 1.5em;
+    --width: calc(var(--thumb-size) * 3);
+  }
+
+  wa-switch::part(thumb) {
+    --border-width: 0;
+    box-shadow: none;
+  }
+
+  wa-switch[checked]::part(thumb) {
+    box-shadow: 0 2px 2px var(--wa-color-brand-40);
+  }
+
+  /* preventing shadows on all form input types */
+  wa-input:not(:focus),
+  wa-input:not([filled]):not([disabled]),
+  wa-checkbox:not([checked]):not([indeterminate]),
+  wa-select:not([filled]):not([disabled]) {
+    --box-shadow: var(--wa-shadow-level-0);
+  }
+
+  wa-checkbox[checked] {
+    --box-shadow: var(--wa-shadow-level-1);
+    --background-checked: linear-gradient(
+      180deg,
+      var(--wa-color-brand-40) 0%,
+      var(--wa-color-brand-50) 51.88%,
+      var(--wa-color-brand-60) 100%
+    );
+  }
+
+  wa-button {
+    --button-gradient-top: var(--wa-color-base-40);
+    --button-gradient-middle: var(--wa-color-base-50);
+    --button-gradient-bottom: var(--wa-color-base-60);
+    --button-shadow-outer: var(--wa-color-base-60);
+    --button-shadow-inner-a: var(--wa-color-base-30);
+    --button-shadow-inner-b: var(--wa-color-base-80);
+    --button-text-shadow: var(--wa-color-base-40);
+    --background: linear-gradient(
+      180deg,
+      var(--button-gradient-top) 0%,
+      var(--button-gradient-middle) 51.88%,
+      var(--button-gradient-bottom) 100%
+    );
+    --box-shadow: 0px 6px 16px var(--button-shadow-outer), inset 0 -2px 8px var(--button-shadow-inner-a),
+      inset 0 2px 8px var(--button-shadow-inner-b);
+    --border-color: var(--button-gradient-middle);
+    --border-width: 1px;
+    --background-hover: linear-gradient(
+      180deg,
+      var(--button-gradient-bottom) 0%,
+      var(--button-gradient-middle) 51.88%,
+      var(--button-gradient-top) 100%
+    );
+    --border-color-hover: var(--button-gradient-middle);
+    --background-active: linear-gradient(
+      180deg,
+      color-mix(in oklab, var(--button-gradient-top), var(--wa-color-mix-active)) 0%,
+      color-mix(in oklab, var(--button-gradient-middle), var(--wa-color-mix-active)) 51.88%,
+      color-mix(in oklab, var(--button-gradient-bottom), var(--wa-color-mix-active)) 100%
+    );
+    --border-color-active: var(--button-gradient-middle);
+
+    &::part(base) {
+      padding-inline-start: var(--wa-space-s);
+    }
```

```suggestion
    &::part(prefix) {
      padding-inline-start: calc(0.25em - var(--border-width));
    }
    &::part(suffix),
    &::part(caret) {
      padding-inline-end: calc(0.25em - var(--border-width));
    }
```

This was causing any button without a prefix icon to look off-center. To even things out, how about we add a bit of padding to the `prefix` and `suffix` slots? I used `em` units so that we don't have to declare separate rules per button size, and I accounted for the button's `--border-width` so that `outline` buttons don't get unnecessarily wide.

(Side note: good heavens, we _have_ to make it easier to change button padding. 😵‍💫 Took me far too long to figure this out...)

---

OH! Yes I noticed that once i made the change but forgot to change it. Thanks for catching it!